### PR TITLE
Compatibility with the Random Content Mod

### PR DESCRIPTION
**Pull request by CarbonY26**
Issue #732

### CODE REVIEW

**Comment by CarbonY26** [comment]
Why did it separate the PR into 2?

**Comment by CarbonY26** [comment]
Now for the flags and other stuff

File: EU4toV2/Data_Files/blankMod/output/common/countries.txt [hunk 1]
```diff
@@ -346,8 +346,9 @@ DMS		="countries/Desmond.txt"
 ENL		="countries/England.txt"#
 KID		="countries/Kildare.txt"
 HSC		="countries/Highlands.txt"
-LAN		="countries/Lancaster.txt"##
 LEI		="countries/Leinster.txt"##
+LNC		="countries/Lancaster.txt"##
```
Comment (IohannesIohannium) [inline EU4toV2/Data_Files/blankMod/output/common/countries.txt:350]: You can remove both LNC and LOI lines as they're not strictly needed. If the output has Lancaster, it'll be correctly set already; if not, there is no need for it, as we summon not these tags in decisions or events.
If you want to keep them, however, you must add the relevant localisations on 0_Names.csv

**Review by IohannesIohannium — CHANGES_REQUESTED** [review]
Missing files needed: LNC_monarchy.tga, LNC_republic.tga, LNC_communism.tga and LNC_fascism.tga
All of these, along with LNC.tga, should be then copied in Vic2toHoi4

**Comment by CarbonY26** [comment]
Going to restructure this PR.

**Pull request closed without merging**